docs/UserGuide.md

```diff
@@ -286,19 +286,24 @@ excuted.
 
 Delete a useless floating task/event/deadline on TaskTracker.
 
-    del "TASK_NAME"
+    del TASK_UNIQUE_INDEX
 
-* Task name should be in a pair of quotation marks.
+* `TASK_UNIQUE_INDEX` will be different from each to each single task.
```

Comment:
For the task's unique index, we could instead use a format like "Day.Month.Year.Number". For example, if I'm entering a 4th task on 3rd May 2016, the task's unique index will be generated as: "3M164".....or "3.M.16.4"...
This way the upper bound needs to be set only for a specific day....which is much easier than setting an upper bound for the whole period when the user is using the app
Also, it can be easier to sort tasks based on days